apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -108,7 +108,39 @@ const linkingOptions = {
       : null;
   },
 
-  prefixes: ["ledgerlive://", "https://ledger.com"],
+  prefixes: [
+    "ledgerlive://",
+    "https://ledger.com",
+    /**
+     * Adjust universal links attached to iOS Bundle ID com.ledger.live
+     * (local debug, prod & nightly builds)
+     * (https://r354.adj.st/.well-known/apple-app-site-association)
+     * (https://ledger.go.link/.well-known/apple-app-site-association)
+     *
+     * to use these universal links use this parameter: adj_t=r1guxhk
+     *  */
+    "https://r354.adj.st",
+    "https://ledger.go.link",
+    /**
+     * Adjust universal links attached to iOS Bundle ID com.ledger.live.debug
+     * (https://a4wc.adj.st/.well-known/apple-app-site-association)
+     * (https://ledger-debug.go.link/.well-known/apple-app-site-association)
+     *
+     * to use these universal links use this parameter: adj_t=f1vrzvp
+     *  */
+    "https://a4wc.adj.st",
+    "https://ledger-debug.go.link",
+    /**
+     * Adjust universal links attached to iOS Bundle ID com.ledger.live.dev
+     * (staging builds)
+     * (https://fvsc.adj.st/.well-known/apple-app-site-association)
+     * (https://ledger-staging.go.link/.well-known/apple-app-site-association)
+     *
+     * to use these universal links, use this query parameter: adj_t=p72sbdr
+     *  */
+    "https://fvsc.adj.st",
+    "https://ledger-staging.go.link",
+  ],
```

Comment:
Implement support for deep links coming from all universal links domains, in `react-navigation` linking configuration.